app/components/support_interface/audit_trail_component.html.erb

```diff
@@ -1,16 +1,17 @@
+<% @pagy, @audits = audits %>
```

Comment:
Can this be moved into the component rb file. Try keeping assignments in the ruby class rather than the view